### PR TITLE
Logic clears inputs & sends message, responsive too.

### PR DESCRIPTION
**Pull request by KyleJayMaxwell**
Thought I committed before the i did the logic of JS. Oh well

### CODE REVIEW

**Comment by KyleJayMaxwell** [comment]
Accidentally forgot to answer the question.

File: main.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@media only screen and (max-device-width: 600px){
```
Comment (bwreid) [inline main.css:1]: Everything inside of here is not actually getting applied!

File: main.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+$(document).ready(function(){
+  
+  $('button').on('click', function(el){
```
Comment (bwreid) [inline main.js:3]: You want to add the event listener to the form itself -- right now, this code is running each time you click the button. That means you don't have to enter anything in for the form to return the alert.

File: readme.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ You will be building a responsive login page based on the mockup found in the [m
 1. Make a Pull Request against this repository
 
 __Discuss the relationship between HTML, JavaScript, the DOM API, and jQuery:__
-> Write your answer here...
+* I think HTML is the base or structure of a website, just a Static website. JS and Jquery are what make the website interactive, ie the clicks and different motions that happens on a website, pretty much the thing everyone wants on their site. DOM and API, DOM is how we can use JS to manipulate the HTML or website in the positions. API is a call from the cloud, filled with data we can use on our website, navigating the tools of JS/Jquery to then use the DOM to put the spice on the HTML.
```
Comment (bwreid) [inline readme.md:21]: I think your explanation of the DOM API is a bit muddled here. When you use JavaScript to change the page, are you interacting with the DOM or HTML? Why?

**Comment by bwreid** [comment]
Take a look at my comments and your code. It looks as though you'll need to change a few things to get the page to accurately work!

**Pull request closed without merging**